runelite-client/src/main/java/net/runelite/client/plugins/loginscreen/LoginScreenPlugin.java

```diff
@@ -243,43 +251,64 @@ private void overrideLoginScreen()
 	{
 		client.setShouldRenderLoginScreenFire(config.showLoginFire());
 
-		if (config.loginScreen() == LoginScreenOverride.OFF)
-		{
-			restoreLoginScreen();
-			return;
-		}
-
 		SpritePixels pixels = null;
-		if (config.loginScreen() == LoginScreenOverride.CUSTOM)
+		Random random = new Random(System.nanoTime());
```

Comment:
You don't need to seed random.

Reply:
Ok I will remove I just saw that is was done in [WorldSupplier.java](https://github.com/runelite/runelite/blob/5494249636322830e2672da905bd1e1eec81a53f/runelite-client/src/main/java/net/runelite/client/rs/WorldSupplier.java) so I did it to be safe